src/factories/interfaces/IPIM_WorkflowFactory_v1.sol

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+// Internal Interfaces
+import {IOrchestrator_v1} from
+    "src/orchestrator/interfaces/IOrchestrator_v1.sol";
+import {IModule_v1} from "src/modules/base/IModule_v1.sol";
+import {IInverterBeacon_v1} from "src/proxies/interfaces/IInverterBeacon_v1.sol";
+import {IOrchestratorFactory_v1} from
+    "src/factories/interfaces/IOrchestratorFactory_v1.sol";
+import {IFM_BC_Bancor_Redeeming_VirtualSupply_v1} from
+    "@fm/bondingCurve/interfaces/IFM_BC_Bancor_Redeeming_VirtualSupply_v1.sol";
+import {IPIM_WorkflowFactory_v1} from
+    "src/factories/interfaces/IPIM_WorkflowFactory_v1.sol";
+import {IERC20Issuance_v1} from "src/external/token/IERC20Issuance_v1.sol";
+
+// Internal Dependencies
+import {ERC20Issuance_v1} from "src/external/token/ERC20Issuance_v1.sol";
+
+// External Interfaces
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+
+interface IPIM_WorkflowFactory_v1 {
+    //--------------------------------------------------------------------------
+    // Errors
+    /// @notice Error thrown when an unpermissioned address tries to claim fees or to transfer role.
+    error PIM_WorkflowFactory__OnlyPimFeeRecipient();
+    /// @notice Error thrown when the curve is deployed with an invalid configuration.
+    error PIM_WorkflowFactory__InvalidConfiguration();
+
+    //--------------------------------------------------------------------------
+    // Events
+
+    /// @notice Event emitted when a new PIM workflow is created.
+    /// @param bondingCurve The address of the bonding curve.
+    /// @param issuanceToken The address of the issuance token.
+    /// @param deployer The address of the deployer.
+    /// @param recipient The address of the recipient.
+    /// @param isRenouncedIssuanceToken If ownership over the issuance token should be renounced.
+    /// @param isRenouncedWorkflow If admin rights over the workflow should be renounced.
+    event PIMWorkflowCreated(
+        address indexed bondingCurve,
+        address indexed issuanceToken,
+        address indexed deployer,
+        address recipient,
+        bool isRenouncedIssuanceToken,
+        bool isRenouncedWorkflow
+    );
+
+    /// @notice Event emitted when factory owner sets new fee.
+    /// @param oldRecipient The previous pim fee recipient.
+    /// @param newRecipient The new pim fee recipient.
+    event PimFeeRecipientUpdated(
+        address indexed oldRecipient, address indexed newRecipient
+    );
+
+    /// @notice Event emitted when PIM fee (buy/sell fees) is claimed.
+    /// @param claimer The address of the claimer.
+    /// @param amount The amount claimed.
+    event PimFeeClaimed(address indexed claimer, uint amount);
```

Comment:
General: In our `LM_PC_Bounties_v1` we use "claimant", which is a synonym of "claimer", but I'd prefer to be consistent, so let's use "claimant" here as well imho.